Assets/HoloToolkit/Input/Scripts/InputSources/BaseInputSource.cs

```diff
@@ -11,88 +11,11 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public abstract class BaseInputSource : MonoBehaviour, IInputSource
     {
-        public event EventHandler<InputSourceEventArgs> SourceUp;
-        public event EventHandler<InputSourceEventArgs> SourceDown;
-        public event EventHandler<SourceClickEventArgs> SourceClicked;
-        public event EventHandler<InputSourceEventArgs> SourceDetected;
-        public event EventHandler<InputSourceEventArgs> SourceLost;
-        public event EventHandler<HoldEventArgs> HoldStarted;
-        public event EventHandler<HoldEventArgs> HoldCompleted;
-        public event EventHandler<HoldEventArgs> HoldCanceled;
-        public event EventHandler<ManipulationEventArgs> ManipulationStarted;
-        public event EventHandler<ManipulationEventArgs> ManipulationUpdated;
-        public event EventHandler<ManipulationEventArgs> ManipulationCompleted;
-        public event EventHandler<ManipulationEventArgs> ManipulationCanceled;
-        public event EventHandler<NavigationEventArgs> NavigationStarted;
-        public event EventHandler<NavigationEventArgs> NavigationUpdated;
-        public event EventHandler<NavigationEventArgs> NavigationCompleted;
-        public event EventHandler<NavigationEventArgs> NavigationCanceled;
-        public event EventHandler<SpeechKeywordRecognizedEventArgs> SpeechKeywordRecognized;
-
-        public abstract SupportedInputEvents SupportedEvents { get; }
-        
-        private bool isRegistered = false;
-
+        protected InputManager inputManager;
+  
         protected virtual void Start()
         {
-            // Must register on start, because the InputManager might not be initialized when 
-            // registering in OnEnable
-            RegisterWithInputManager();
-        }
-
-        protected virtual void OnDestroy()
-        {
-            UnregisterFromInputManager();
-        }
-
-        protected virtual void OnEnable()
-        {
-            RegisterWithInputManager();
-        }
-
-        protected virtual void OnDisable()
-        {
-            UnregisterFromInputManager();
-        }
```

Comment:
I see you did that with SpeechInputSource, it probably just needs to be added to GesturesInput